script.js

```diff
@@ -28,13 +460,43 @@ jQuery(function () {
         e.stopPropagation();
         return false;
     });
+    
+    var $admin_toggle_btn = jQuery('#tagging__edit_toggle_admin')
+                                .checkboxradio()
+                                .click(function(){
+                                    jQuery('div.plugin_tagging_edit ul.tagging_cloud a').editable('toggleDisabled');
+                                }),
+        add_editable = function() {
+            //no editable button - we are not the admin
+            if ($admin_toggle_btn.length === 0) return;
+            
+            jQuery('div.plugin_tagging_edit ul.tagging_cloud a')
+                .editable({
+                    disabled  : !$admin_toggle_btn[0].checked,
+                    label     : LANG.plugins.tagging.admin_change_tag,
+                    url       : DOKU_BASE + 'lib/exe/ajax.php?call=plugin_tagging_admin_change',
+                    params    : { 'call'   : 'plugin_tagging_admin_change',
+                                  'id'     : JSINFO.id,
+                                  'sectok' : JSINFO.sectok
+                                },
+                    success   :
+                        function(response, newValue) {
+                            jQuery('div.plugin_tagging_edit ul.tagging_cloud').html(response.html_cloud);
+                            $form.find('input[type="text"]').val(response.tags_edit_value);
+                            add_editable();
+                        }
+                });
+        };
+    
+    add_editable();
```

Comment:
The popup is occasionally placed in a way that is not optimal, i.e. outside of the right boundary of the page. 
![screenshot-2017-9-11 start testwiki2](https://user-images.githubusercontent.com/7372507/30268592-843e4a14-96e6-11e7-8dc8-06b6f2dcefec.png)

Would it be possible to use jQueryUI to improve the placement? See its usage for the dokuwiki footnote tooltips:

https://github.com/splitbrain/dokuwiki/blob/2a85c6914dc75522efc5564694de193ddafcb94b/lib/scripts/page.js#L72-L76